llvm/lib/Transforms/Scalar/DeadStoreElimination.cpp

```diff
@@ -2132,182 +2177,182 @@ struct DSEState {
     }
     return MadeChange;
   }
-};
 
-static bool eliminateDeadStores(Function &F, AliasAnalysis &AA, MemorySSA &MSSA,
-                                DominatorTree &DT, PostDominatorTree &PDT,
-                                const TargetLibraryInfo &TLI,
-                                const LoopInfo &LI) {
-  bool MadeChange = false;
+  // Try to eliminate dead defs killed by `KillingLocWrapper` and return the
```

Comment:
nit:  "eliminate dead defs killed by `KillingLocWrapper`" I'm not sure killed by KillingLocWrapper is as clear. Which part of KillingLocWrapper are the defs killed by? It's not the MemoryLocation parts right?

Reply:
Updated to "Try to eliminate dead defs that access `KillingLocWrapper.MemLoc` and are killed by `KillingLocWrapper.MemDef`".